page.go

```diff
@@ -153,6 +153,16 @@ func (p *Page) SetUserAgent(req *proto.NetworkSetUserAgentOverride) error {
 	return req.Call(p)
 }
 
+// SetBlockedURLs For some requests that do not want to be triggered, such as some dangerous operations, delete, quit logout, etc.
+// Wildcards ('*') are allowed,such as ["*/api/logout/*","delete",].
+// NOTE: if you set this pattern `""` it will block all requests
```

Comment:
```suggestion
// Wildcards ('*') are allowed, such as ["*/api/logout/*","delete"].
// NOTE: if you set empty pattern "", it will block all requests.
```